homeassistant/components/sensor/yr.py

```diff
@@ -185,7 +187,8 @@ def try_again(err: str):
                 model = self.data['meta']['model']
                 if '@nextrun' not in model:
                     model = model[0]
-                self._nextrun = dt_util.parse_datetime(model['@nextrun'])
+                self._nextrun = dt_util.parse_datetime(model['@nextrun']) +\
```

Comment:
👍 

At this point we are already between 1 min up to 10 minutes past the `_nextrun` (since they are on the hour) so I think we are ok.

If you want to do something, the simplest would be to randomize between 5 min and 55 min.